server/service/src/sync/sync_buffer.rs

```diff
@@ -23,6 +23,7 @@ const TRANSLATION_AND_INTEGRATION_ORDER: &[&str] = &[
     LegacyTableName::LIST_MASTER_NAME_JOIN,
     LegacyTableName::REPORT,
     LegacyTableName::LOCATION,
+    LegacyTableName::BARCODE,
```

Comment:
oops. this needs to be before the ITEM_LINE as that table now has `barcode_id` being pulled through